### PR TITLE
Port and message set variable

### PR DESCRIPTION
**Pull request by mdumontet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ response in Terminal1 : {"Message":"hello world","ID":57,"Date":"Thu Oct  1 11:0
 # Test with image
 ``` bash 
 for now the port is hard coded in the image
-docker run -d -p 9080:9080 crow/apps/server:v1 9080 yourMessage
-docker run --net=host --rm  crow/apps/client:v1 9080
+docker run -d -p 9080:9080 crow/apps/server:server_image
```
Comment (dcorbin83) [inline README.md:91]: still 9080:9080 ?

File: apps/client/src/client.cpp [hunk 2]
```diff
@@ -4,33 +4,37 @@
...
     {
         isLooping = true;
-        aMsec = atoi(argv[2]);
+        aMsec = atoi(std::getenv("SERVER_MESSAGE"));
     }
 
     std::cout << "query send on " << url << std::endl;
     auto curl = curl_easy_init();
     curl_easy_setopt(curl, CURLOPT_URL, url.c_str());
+    unsigned int debug_max_loop = 0;
```
Comment (dcorbin83) [inline apps/client/src/client.cpp:33]: my loop :(

File: apps/client/BUILD [hunk 3]
```diff
@@ -15,7 +16,11 @@ container_image(
     repository = "crow",
     labels = {
         "com.crow.client": "v1"
-    }
+    },
+    env = {
+        "SERVER_PORT": SERVER_PORT,
+        "SERVER_MESSAGE": SERVER_MESSAGE,
```
Comment (dcorbin83) [inline apps/client/BUILD:22]: I guess SERVER_MESSAGE for the client BUILD

File: README.md (hunk 1 above)
Comment (mdumontet) [reply README.md:91]: I added 9080 has value of the port in the var.bzl so yes except if you change the variable

File: apps/client/src/client.cpp (hunk 2 above)
Comment (mdumontet) [reply apps/client/src/client.cpp]: updated don't cry

**Review by dcorbin83 — APPROVED** [review]

**Pull request merged**